fs/fs.go

```diff
@@ -58,12 +60,15 @@ type RealFsInfo struct {
 	labels map[string]string
 
 	dmsetup dmsetupClient
+
+	thinPoolWatcher *volume.ThinPoolWatcher
```

Comment:
Another option is to define a new device mapper usage object and inject that into the fs handler if required.